eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/email/MailGunEmailService.java

```diff
@@ -0,0 +1,47 @@
+package hu.flowacademy.companycalendar.email;
+
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+@Service
+@RequiredArgsConstructor
+public class MailGunEmailService implements EmailService {
+
+    @NonNull
+    private RESTClient springRestClient;
+    @Value("${mailgun.api.username}") String username;
```

Comment:
Ehhez szerintem vegyünk fel egy külön osztályt, amit felannotálunk `@ConfigurationProperties`-zel, és összekötjük a fieldjeit a külső propertykkel.
Hasznos olvasmány: https://www.baeldung.com/configuration-properties-in-spring-boot